File: web-exam-system-vue/src/api/StudentAnswers.js
import request from "@/util/request";
export function addStudentAnswer(data) {
  return request.post("/studentAnswers/addStudentAnswers", data);
}

export function checkAnswerRecord(studentId, examId) {
  return request.get("/studentAnswers/checkAnswerRecord", {
    params: {
      studentId,
      examId,
    },
  });
}

File: exam-system/src/main/java/cn/com/xuroot/entity/StudentAnswers.java
package cn.com.xuroot.entity;

import lombok.Data;

import java.util.Date;

@Data
public class StudentAnswers {
    private Integer recordId;
    private Integer examId;
    private Integer studentId;
    private Integer questionId;
    private String answer;
    private Integer isCorrect;
    private Integer isReviewed;
    private Integer score;
    private String reviewedBy;
    private String reviewNotes;
    private Date createdAt;
}
